README.md

```diff
@@ -885,8 +885,11 @@ You can specify the debug server address with the `DEBUG_HOST` and `DEBUG_PORT`
 # Local Cache
 
 Ratelimit optionally uses [freecache](https://github.com/coocood/freecache) as its local caching layer, which stores the over-the-limit cache keys, and thus avoids reading the
-redis cache again for the already over-the-limit keys. The local cache size can be configured via `LocalCacheSizeInBytes` in the [settings](https://github.com/envoyproxy/ratelimit/blob/master/src/settings/settings.go).
+redis cache again for the already over-the-limit keys. 
+Cache configuration options can be set using the following environment variables:
+1. The local cache size can be configured via `LocalCacheSizeInBytes` in the [settings](https://github.com/envoyproxy/ratelimit/blob/master/src/settings/settings.go).
 If `LocalCacheSizeInBytes` is 0, local cache is disabled.
+2. To control the behavior of cache keys when the overall status is over-limited, set the `INCREMENTING_CACHE_KEYS_FOR_OVER_LIMIT` variable to "false". This is particularly useful when utilizing multiple descriptors for a group of routes. Note that for this configuration to work, the local cache must be enabled.
```

Comment:
Sorry from this comment as a user I don't know what this does or why I would set it. Can you expand? I'm still not convinced this isn't just a bug fix. Does it actually need a config flag? What would be the situation in which someone would want to keep this as true?

Reply:
Hi @mattklein123,

We don't actually need the config. I added that config only to keep the current behavior as it is (assuming that someone else has depended on that behavior, although it is a bug). But I also believe we don't need this config and we can directly fix this in the code below. Shall I do it?

We can directly execute the code https://github.com/envoyproxy/ratelimit/pull/430/files#diff-cf4d55b6b42859dee0ecf1d127327f14a09781d93dc53cd6267019cb191c870cR61.

Thank you

Reply:
Hi @mattklein123 ,

Just fixed the bug from c39e74da49776669ce0b74b5c5d0b9926a8f52e8

Thank you